tic-tac-toe.js

```diff
@@ -1,7 +1,37 @@
-function TicTacToe() {
-
+var TicTacToe = function() {
+  this.turn = 0;
+  this.board = ["", "spot 1", "spot 2", "spot 3", "spot 4", "spot 5", "spot 6", "spot 7", "spot 8"]
 }
 
-TicTacToe.prototype = {
-  
+  TicTacToe.prototype.dawg = function(cell){
+
+      if(this.turn % 2 == 0){
+        $(cell).addClass("purple-rain");
+      } else {
+        $(cell).addClass("golden-gardens");
+      }
+      this.turn = this.turn + 1;
+    }
+
+  TicTacToe.prototype.
```

Comment:
Code from here on out is… incomplete? That's OK. I just can't tell if it's a work in progress (WIP) or abandoned. Leave comments to remember your state of mind. Delete code you're not using.